### ISSUE TITLE
feat (github): github_repository_collaborators

### ISSUE THREAD

**Issue by chr-b** [cloudquery/cloudquery#12680]
### Which problem is this feature request solving?

The list of collaborators that have access to a repository is currently missing in the plugin.
The table `github_repositories` contains a `collaborators_url` column, but not the actual collaborators.

### Describe the solution you'd like

A new table `github_repository_collaborators` should be added that contains at least the following information:
* User ID (`login`)
* permissions

The `login` column will allow joining with the `login` column from the `github_organization_members` table.

The collaborator data is available in the API `/repos/{owner}/{repo}/collaborators` (see [Github documentation page](https://docs.github.com/en/free-pro-team@latest/rest/collaborators/collaborators?apiVersion=2022-11-28#list-repository-collaborators)).

### Pull request (optional)

- [ ] I can submit a pull request

**Comment by akash1810** [comment]
Adding my 👍🏽 to this, with a description of my use-case.

In my organisation, we've [defined](https://github.com/guardian/recommendations/blob/main/github.md#access) best practice as providing repository access exclusively to teams, avoiding direct individual access. This is because individual access to a repository is retained should you be removed from a GitHub organisation. With team-only based access, removing someone from the GitHub organisation will also remove them from any teams, and thus removes repository access.

A `github_repository_collaborators` table could be joined with `github_team_members`, and `github_team_repositories` to provide a view of repositories with direct individual access. Further joining with `github_organization_members` would show which repositories are accessible by unintended actors.

**Comment by akash1810** [comment]
With this marked as "completed", do we know which version we'll see this data in? There doesn't appear to be a new [release](https://hub.cloudquery.io/plugins/source/cloudquery/github/latest/tables), and I can't find a [PR that relates to this issue](https://github.com/cloudquery/cloudquery/pulls?q=is%3Apr+is%3Aclosed+label%3Aarea%2Fplugin%2Fsource%2Fgithub); maybe the issue was accidentally closed?

Without knowing who to tag cc @pilvikala @erezrokah as you've triaged the issue in the past 😅.

**Comment by erezrokah** [comment]
Hi @akash1810, sorry for the confusion our release schedule is on Tuesday will release this today

**Comment by akash1810** [comment]
> Hi @akash1810, sorry for the confusion our release schedule is on Tuesday will release this today

Ah, amazing! Thanks! I'm a bit excited by this change, as you can tell 😄.

**Comment by erezrokah** [comment]
This is released with GitHub `v10.0.0` https://hub.cloudquery.io/plugins/source/cloudquery/github/v10.0.0/versions

@akash1810 you might also be interested in a new table option we added:
```yaml
kind: source
spec:
 ...
  spec:
     table_options:
       github_workflow_runs:
         created_since: "3 days agon"
```

To limit the number of workflow runs in a sync (to avoid hitting rate limits)
